### PR TITLE
Fix build errors for minizip

### PR DESCRIPTION
**Pull request by greg-hellings**
Fixes build errors related to minizip - doesn't seem to write the files properly, though.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,6 +46,7 @@ requires:
     - intltool
     - libgsf
```
Comment (alerque) [inline .travis.yml:47]: Besides adding minizip, libgsf should be removed from the dependencies for all these builds just to check that it works without it.

File: .travis.yml [hunk 2]
```diff
@@ -141,6 +144,7 @@ requires:
     - libglade2-dev
     - libgsf-1-dev
```
Comment (alerque) [inline .travis.yml:145]:
```suggestion
```

Comment (greg-hellings) [reply .travis.yml]: Good call. I've removed all the gsf's from the build deps. New test up.

**Comment by alerque** [comment]
Ooops, sorry. I had those code reviews written earlier but they didn't post for some reason. I guess we can disregard ... except that zlib is now also a dependency for all platforms. I'm not too worried given in is pretty ubiquitous.

**Pull request merged**